suite-native/device-manager/src/components/DeviceItem/SimpleDeviceItemContent.tsx

```diff
@@ -0,0 +1,61 @@
+import { HStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { ConnectionDot } from './ConnectionDot';
+import { SwitchContentProps } from '../../types';
+
+const headerStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+    overflow: 'visible',
+}));
+
+export const SimpleDeviceItemContent = ({
+    device,
+    headerTextVariant,
+    isPortfolioLabelDisplayed,
+    header,
+    isPortfolioTrackerDevice,
+}: SwitchContentProps) => {
```

Comment:
`SwitchContentProps` define `subheader` prop that is never used here. It should not be possible to send a irelevant prop to a component. Please fix this.

Reply:
4650a6feb3